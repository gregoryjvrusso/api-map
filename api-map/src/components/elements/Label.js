import styled, { css } from 'styled-components'
import { theme, ifProp } from 'styled-tools'
import { fontSize, space } from 'styled-system'

const warning = css`
  color: ${p => p.theme.colors.texts.warning};
`

const Label = styled.label`
  padding: 8px 8px;
  margin: 2px 10px;
  color: ${theme('color', 'yellow')};
  ${ifProp('warning', warning)}
  ${fontSize}
  ${space}
`

export default Label
