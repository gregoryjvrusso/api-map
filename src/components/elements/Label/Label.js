import styled from 'styled-components'

const Label = styled.label`
  padding: 8px 8px;
  margin: 2px 10px;
`

export default Label
