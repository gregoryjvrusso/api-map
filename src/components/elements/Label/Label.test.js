import React from 'react'
import { render } from '@testing-library/react'
import Label from './Label.js'

describe('Label component', () => {
  it('deve conseguir renderizar [snapshot]', async () => {
    const { container } = render(<Label />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
