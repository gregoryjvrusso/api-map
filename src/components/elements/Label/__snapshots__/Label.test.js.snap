// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label component deve conseguir renderizar [snapshot] 1`] = `
<label
  class="sc-bdVaJa FTQhB"
/>
`;
